annotations.Rmd

```diff
@@ -5,15 +5,27 @@ source("common.R")
 columns(1, 2 / 3)
 ```
 
-## Labels {#sec:labelling}
-\index{Labels} \index{Text} \indexf{geom\_text}
+When constructing a data visualisation, it is often necessary to make annotations to
+the data displayed. Conceptually, an annotation supplies *metadata* for the plot: that is,
+it provides additional information about the data being displayed. From a practical 
+standpoint, however, metadata is just another form of data. Because of this, the 
+annotation tools in ggplot2 reuse the same geoms that are used to create other plots. 
+However, to meet the specific needs that users often have when annotating plots, there
+are some helper functions in ggplot2 itself, and a number of other packages have 
+extended ggplot2 in ways you may find helpful. 
+
+\index{Metadata}
 
-Adding text to a plot can be quite tricky. ggplot2 doesn't have all the answers, but does provide some tools to make your life a little easier. The main tool is `geom_text()`, which adds `label`s at the specified `x` and `y` positions. 
 
-`geom_text()` has the most aesthetics of any geom, because there are so many ways to control the appearance of a text:
+## Adding Text Labels to Points {#sec:labelling}
```

Comment:
Can you please use sentence case in headings?

And maybe just call "Text labels"?